hooks/useAuthStore.ts

```diff
@@ -36,11 +36,20 @@ export const useAuthStore = create<AuthState>((set) => ({
     try {
       set({ isLoading: true, error: null });
       const { data, error } = await supabase.auth.signInWithPassword({ email, password });
-      if (error) throw error;
+      if (error) {
+        console.log("Signin error:", error.message)
+        let errorMessage = 'Failed to sign in';
+        if (error.message.includes('Invalid login credentials')) {
+          errorMessage = 'Invalid email or password. Please try again.';
```

Comment:
Hmm I'm guessing you were trying to change the error message here for the user's benefit. I think this is something we can revisit and for now it's best we see what the actual `error.message` is from Supabase in the UI.

We should be throwing the error on line 47 with `error.message`

It would be nice if we could surface this error in the UI on the sign in page (also for sign up)

Reply:
Yes that was my thinking. So for now I will simplify and use the direct message as you suggested.